pkg/config/config_template.yaml

```diff
@@ -878,6 +878,16 @@ api_key:
     #
     # oid_batch_size: 5
 
+{{- if (eq .OS "windows")}}
+  ## @param windows_counter_refresh_interval - number - optional - default: 60
+  ## This controls the interval in seconds for which windows performance counters are refreshed by the agent.
+  ## Refreshes are necessary for the discovery of new counters/countersets and instances.
+  ## Refresh can be disabled by setting the interval to 0.
```

Comment:
This should be more explicit about the fact that it's used by Agent integrations, and ideally mention what integrations use this option.

Reply:
Thanks for the feedback. I've added a better description. Please let me know if it needs to be even more detailed. 🙇